data-raw/make_cluster_vcov.R

```diff
@@ -0,0 +1,50 @@
+
+# Create clustered variance-covariance matrix for package tests
+
```

Comment:
Is there a citation or something? Doesn't have to be but would be nice, I think.